src/main/java/app/model/Map.java

```diff
@@ -257,4 +257,27 @@ private boolean clearSpot(Vector v)
         }
         return true;
     }
+
+
+    public void capture(Agent currentAgent)
+    {
+        if(currentAgent.getTeam() == Team.GUARD)
+            for(Agent otherAgent: agents){
+                if (otherAgent.getTeam() !=currentAgent.getTeam()){
+                    double dis = currentAgent.getPosition().dist(otherAgent.getPosition());
```

Comment:
`dis` should be renamed to `dist`; this is the most common abbreviation used, especially as this is the name of the method you are calling (Vector.dist())  `dis` is not.